nixos/modules/security/hidepid.nix

```diff
@@ -0,0 +1,32 @@
+{ config, lib, ... }:
+with lib;
+
+{
+  options = {
+    security.hideProcessInformation = {
+      enable = mkEnableOption = ''
```

Comment:
Indeed it is :) I have yet to actually test this implementation, I'll hold off on doing so until I've tested the waters, to minimize wasted effort.